### DISCUSSION TITLE
I can't change the language to Chinese in settings

### DISCUSSION THREAD

**Discussion by xiaoua** [super-productivity/super-productivity#3079] [Q&A] [answered]
There is a Chinese language pack file in the software, but I cannot find the corresponding option in the language settings of the software. How should I change it?

**Comment by johannesjo** [comment, marked as answer]
Hey there! This is a bug currently as the selection should be scrollable. As a workaround until the next release, you can zoom out of the page via strg + scroll wheel to make all entries appear (chinese is at the very bottom due the language code beginning with z).
